capa/ida/plugin/proxy.py

```diff
@@ -92,7 +92,7 @@ def filter_accepts_row_self(self, row, parent):
         @param parent: QModelIndex of parent
         """
         # filter not set
-        if self.min_ea is None and self.max_ea is None:
+        if self.min_ea is None or self.max_ea is None:
```

Comment:
we're currently not handling the case of only one of `min_ea` or `max_ea` being set. im not sure if thats possible, so ive made this guard more restrictive.